### PR TITLE
Qt 5.15.6

### PR DESCRIPTION
**Pull request by hmaarrfk**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -79,11 +84,3 @@ about:
...
   recipe-maintainers:
     - conda-forge/qt-main
-    - ccordoba12
-    - duncanmmacleod
-    - gillins
-    - mingwandroid
-    - msarahan
-    - ocefpaf
-    - stuarteberg
-    - Tobias-Fischer
```
Comment (hmaarrfk) [inline recipe/meta.yaml:89]: These users are all redundant

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,10 +1,15 @@
...
+{% set version = "5.15.6" %}
 
 package:
   name: qt
   version: {{ version }}
 
+# Add a dummy source so that the bot helps us keep this up to date
+source:
+  url: https://download.qt.io/official_releases/qt/{{ version.rpartition('.')[0] }}/{{ version }}/submodules/qtbase-everywhere-opensource-src-{{ version }}.tar.xz
+  sha256: 05edd00b2a1ba99c85b3fe876fe1c23d63f0a9bbca7df52bc47393cfd8c809c7
```
Comment (hmaarrfk) [inline recipe/meta.yaml:11]:
```suggestion
  sha256: 396bc6b0d773ac6a7c691a4c3d901999f571e3e7033d7fd6f65e4ef2b6eb7340
```

**Comment by hmaarrfk** [comment]
@conda-forge-admin please rerender

File: recipe/meta.yaml (hunk 1 above)
Comment (ocefpaf) [reply recipe/meta.yaml:89]: I was removed from the team with this :-/

Comment (hmaarrfk) [reply recipe/meta.yaml:89]: You should have stayed in. you are listed as part of the qt-main group
https://github.com/conda-forge/qt-main-feedstock/blob/main/recipe/meta.yaml#L253

Comment (hmaarrfk) [reply recipe/meta.yaml:89]: Your membership is likely going to be "implicit".

Comment (ocefpaf) [reply recipe/meta.yaml:89]: Sure. But some folks may need an explicit one for day job reasons. We need to check with the team before doing this.

**Pull request merged**